### ISSUE TITLE
DreadLamp doens't block travelling merchant.

### ISSUE THREAD

**Issue by demethan** [Xalcon/TorchMaster#114]
A new MC feature is that damn traveller merchant with his lamas disgusting lamas.  I really dislike the sound.  Anyways, 

I would think this lamp would block that,  but I assume it require special code to make it work, because it doesn't right now.   

If you could include that at some point I think it would be a nice feature. 

Greatly appreciated. 

Info: pack = ATM5
your mod version 2.1.0-rc0

**Comment by Xalcon** [comment]
Yeah, the wandering trader uses its own spawn system and forge doesnt provide any specialized hooks into it yet. I might be able to still hook into it myself, but not in the same generic way as with mob spawning. I'll investigate this when I find the time.

**Comment by McKitten** [comment]
Please do. 
I am so sick of the merchant trying to enter my house, breaking the leads to his llamas and then them finding a way into my crops.
I do not normally use the dreadlamp, but for this, I may.
